make/mcu/SITL.mk

```diff
@@ -35,6 +35,7 @@ MCU_EXCLUDES = \
             drivers/rx/rx_xn297.c \
             drivers/display_ug2864hsweg01.c \
             telemetry/crsf.c \
+            telemetry/ghst.c \
```

Comment:
The product line is called 'Ghost', the protocol is called 'GHST'. I am happy to tweak the protocol name for Betaflight, but that would cause an inconsistency with OpenTx.